tinynumpy/tinynumpy.py

```diff
@@ -408,7 +408,13 @@ def cross(u, v):
     uxv = []
 
     # http://mathworld.wolfram.com/CrossProduct.html
-    if uDim == vDim == 3:
+    if uDim == vDim == 2:
+        try:
+            uxv.append(0)
```

Comment:
It was there for the 3D case and just assumed it was initialising the array. Now I think about it doesn't make sense in either case so I'll fix it.